app/services/omniauth.rb

```diff
@@ -0,0 +1,25 @@
+class Omniauth
```

Comment:
Since omniauth is a gem....can we call this something else? Maybe `LoginHelpers` `SessionService` or something? This is not something I think we should monkeypatch into another namespace

Reply:
Didn't realize Omniauth was a gem, thanks for the alert.